pkg/trace/transform/transform.go

```diff
@@ -100,6 +102,27 @@ func OtelSpanToDDSpanMinimal(
 	return ddspan
 }
 
+func isDatadogAPMConventionKey(k string) bool {
+	return k == "service.name" || k == "operation.name" || k == "resource.name" || k == "span.type" || k == semconv127.AttributeDeploymentEnvironmentName || k == semconv.AttributeDeploymentEnvironment || k == "http.method" || k == "http.status_code"
+}
+
+func mapHTTPAttributes(k string, value string, ddspan *pb.Span) {
```

Comment:
This func should have a better name: it not only maps HTTP attributes, also sets other meta tags. Maybe call it sth. like setMetaOTLPWithHTTPMapping etc.